### PR TITLE
Remove community ESPHome addon

### PR DESCRIPTION
**Pull request by mbo18**
Update ESPHome addon repository

### CODE REVIEW

File: site/src/analytics-addons.ts [hunk 1]
```diff
@@ -130,8 +130,8 @@ const ADDONS: {
   a0d7b954_esphome: {
     name: "ESPHome Home Assistant Add-On",
     documentation:
-      "https://github.com/hassio-addons/repository/blob/master/esphome/DOCS.md",
-    icon: "https://raw.githubusercontent.com/hassio-addons/repository/master/esphome/icon.png",
+      "https://github.com/esphome/home-assistant-addon/blob/main/esphome/DOCS.md",
```
Comment (ludeeus) [inline site/src/analytics-addons.ts:133]: This is not correct, this is a different addon.
The esphome repo is defined at the bottom of this list.

Comment (mbo18) [reply site/src/analytics-addons.ts:133]: Yeah just saw that. Should this be removed? Icon and link are broken

Comment (ludeeus) [reply site/src/analytics-addons.ts:133]: Probably. If its not in that repo anymore, we should not have it here.

Comment (frenck) [reply site/src/analytics-addons.ts:133]: The add-on has been removed from the community repository (a couple of days ago).

**Review by ludeeus — APPROVED** [review]

**Pull request merged**